app/controllers/idv/enter_password_controller.rb

```diff
@@ -124,8 +124,10 @@ def gpo_mail_service
     end
 
     def init_profile
-      idv_session.create_profile_from_applicant_with_password(password)
-
+      idv_session.create_profile_from_applicant_with_password(
+        password,
+        resolved_authn_context_result.enhanced_ipp?,
+      )
```

Comment:
I like that you pass in the eval of `resolved_authn_context_result.enhanced_ipp?` here. I hard coded it to false in my PR. This is better, I will change my code as a result.